envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadata.kt

```diff
@@ -54,6 +74,33 @@ private fun getCommunicationMode(proto: Value?): CommunicationMode {
     }
 }
 
+fun Value?.toStatusCodeFilter(properties: AccessLogFilterProperties): AccessLogFilter.StatusCodeFilter? {
+    val value = this?.stringValue
+
+    if (value != null) {
+        var op: ComparisonFilter.Op? = null
+        val regex = """((le)|(eq)|(ge)){1}:(\d{3})""".toRegex()
```

Comment:
Won't `toRegex()` make issues here, like a recent memory pressure? I think we should consistently use re2j with a static matcher so we won't make accidentally the same mistake again.

Reply:
I think @slonka fixed that issue.

Reply:
I'll use re2j pattern